input/size.go

```diff
@@ -115,6 +115,7 @@ const (
 	//	- Sequence: 4 bytes
 	InputSize = 32 + 4 + 1 + 4
 
+	// FundingInputSize 41 bytes
```

Comment:
Where's the script that was run? Should we consider adding it to the repo, or instead extending our tests to craft the _actual_ transaction to compare to the estimate?

Reply:
It was a pretty rough script I wrote, not sure if worth adding it since we shouldn't be changing these very often.

Posting for reference: https://github.com/halseth/lnd/commit/d86684ca893553ce582238a6909d9285be0dcdc8

Reply:
my vote would be to add the script and have it generate a test suite that verifies the actual size. we can fail the build if the tests are not properly committed in git or fail the size checks. I'd be okay with making that a follow up as well